### ISSUE TITLE
Initial input is not keyboard accessible (not focusable)

### ISSUE THREAD

**Issue by thomasrye** [lukemurray/NgHierarchicalSelector#22]
Even if `tabindex="0"` is added to the hierarchical-input, it doesn't seem to allow `Enter` to open the tree view. Once the tree view is open (via mouse click), there are keyboard actions to navigate the tree ok.

As it is, this widget is not accessible at all because you can't get to the initial state to open the tree via keyboard.
